index.html

```diff
@@ -56,142 +62,153 @@ <h2>Glad to see you!</h2>
         </div>
             
     </section>
-    <section class="clm-2">
+    <section class="clm-2" id="portfolio">
         <div class="card-container">
             <div class="card card-1">
-                <img src="./images/portfolio_1.png" alt="my first work">
-                <h2>Tonic</h2>
-                <div class="canopy">
-                    <h4>Canopy</h4>
-                    <ul>
-                        <li>Back End Dev</li>
-                        <li>2015</li>
+                <img src="./images/Snapshoot-Portfolio-1.svg" alt="my first work">
+                <div class="details">
+                    <h2>Tonic</h2>
+                    <div class="canopy">
+                        <h4>Canopy</h4>
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="card-descript">A daily selection of privately 
+                        personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="languages">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
                     </ul>
+                    <button>See Project</button>
```

Comment:
From the project requirements, when the button is clicked it should take you to the Live Demo of the project or to the GitHub repository. Please fix this in all the buttons.

Reply:
Hi @keneogu!
we don't have those projects yet and all the buttons are just there for design. I think they will play their roles after completing other projects. The task for today was just to create the mobile and desktop version; it's our first project.
That's why there are no links.